include/opencl_utils.h

```diff
@@ -115,7 +115,7 @@ class OpenCL {
 				sort(times.begin(), times.end());
 				auto median = times[times.size() / 2];
 				// Save result to file
-				File::add_time(run.hash, median, local_size);
+				File::add_time(run.hash, median, cl::NDRange(run.glob1, run.glob2, run.glob3),local_size);
```

Comment:
why do you use `run.X` for the global sizes but `local_size` for the local sizes and not the same approach for both?